src/lib/config.ts

```diff
@@ -1334,6 +1334,15 @@ export class default_config {
      * Whether to show article url in the document.title of Reader View.
      */
     readerurlintitle: "true" | "false" = "false"
+
+    /**
+     *  Which css styles to add for hint elements.
```

Comment:
We should say what fg/bg/outline mean and give an example of how to set it, e.g. `:set hintstyles.bg none`